sequence_processing_pipeline/Pipeline.py

```diff
@@ -133,6 +145,34 @@ class Pipeline:
 
     assay_types = [AMPLICON_ATYPE, METAGENOMIC_ATYPE, METATRANSCRIPTOMIC_ATYPE]
 
+    @staticmethod
+    def make_sif_fname(run_id, full_project_name):
+        # TODO: the problem with this structure is that there's no clear way
+        #  to figure out, for an arbitrary sif fname, where the run id ends and
+        #  where the project name begins because single underscores are used as
```

Comment:
Would a simple fix be to use `:`?

Reply:
Sorry, is the suggestion to use a colon in the sample info file name?  I should maybe clarify that the current sif naming convention is pretty baked in; I don't think it can be easily changed at this point without some considerable review of the code.  This new method therefore isn't really creating a the sif naming convention--it is just *encapsulating* it to move us toward an easier refactoring in the future.  The note here is kind of a reminder to myself for when we arrive at that happy day ... :D